### DISCUSSION TITLE
Prepare a survey for current beta run

### DISCUSSION THREAD

**Discussion by Lineflyer** [cgeo/cgeo#11913] [Ideas]
As discussed around the topic of the new icon set there was the idea to have a survey.

The easiest possibility (which we once did) is to use Google Forms. The downside is that no muli-language is supported, else it was well accepted by users.

Unless someone is willing and able to find and setup another survey tool, I suggest to use Google Forms once again and make a very small survey first.

I could imagine:
- List all icons one by one and ask for a rating with optional free text for a comment
- Final question: How do you like the new icon set overall?

**Comment by Lineflyer** [comment]
This could be a draft (icon pictures missing...need PNGs for that) if we use Google Forms:

![image](https://user-images.githubusercontent.com/949669/137586522-89802d15-2742-4629-b1be-2c8dd3d6ab31.png)
![image](https://user-images.githubusercontent.com/949669/137586504-3b03305a-11a1-48b9-a88e-1a3dd605fb1c.png)
and so on

**Reply by Lineflyer** [reply]
> (icon pictures missing...need PNGs for that) 

Ideally would be some PNG with 64x64 it seems


![image](https://user-images.githubusercontent.com/949669/137587413-62e321cc-21f1-403e-bbe4-9367e5abff2e.png)

**Reply by moving-bits** [reply]
Thanks for preparing the proposal. One thing we should add, though, is a text field per icon to get some indication WHY a specific icon is being loved or not, and what users propose to change for that icon.

**Reply by Lineflyer** [reply]
@moving-bits 
Yes, I understand what you mean. But (as stated below) it will be hard to impossible to review all text fields afterwards. Therefore my idea would be to start with rating and do either a second round for those with lowest rating or provide a free text only if the user provides a low rating (not sure if that is supported with Google forms).

**Reply by ztNFny** [reply]
Looking at the current feedback it's clear which icons are not liked, no need to have some numbers on that.
Also a number will be useless in that regard - what does it mean if someone doesn't like the traditional icon? What problem does he have with it? Contrast too low, bad colors, icon not recognizable, confused the icon with another (see the event feedback), ...? A number doesn't provide any meaningful input in that way.
Such votes could only be used to compare options, like "which of these 2 icons do you like better" - which we already did in the beginning on the page I set up.

**Comment by fm-sys** [comment]
IMHO, the header should use an updated screenshot showing the new cache markers...

**Reply by fm-sys** [reply]
This file will help while updating the graphic:

https://github.com/cgeo/cgeo/blob/master/main/project/playstore/screenshots/aditional%20data/cgeo%20playstore%20images%20-%20functional%20graphic.pptx

**Comment by ztNFny** [comment]
I'm against a one by one 1-5 survey, that doesnt provide any usable feedback on why it's not liked and what to improve.
Instead it should be text fields with some guidance what to fill there. As stated in the other thread, I can provide se input next week.

**Reply by Lineflyer** [reply]
Yep, feel free to make a proposal.
If we however only offer a "free text" it will be hard to evaluate (the survey returns a large list of texts only). 
Therefore I thought to do a click rating first (as this is easy to evaluate) and do a second round of survey for those with low overall ratings and also there rather provide a multiple choice instead of a free text (or maybe a multiple choice with an "Other" option and free text for that one).

**Reply by ztNFny** [reply]
We should also include a question about accessibility in there. Like "are you visually impaired?"
E.g. users with color blindness have different needs than others. And while it's hard to address all such cases we should at least try (or take that as a task for the future, to provide an additional "theme")

**Comment by ztNFny** [comment]
Oh, and I'd like to wait until after beta 2. No point in more "i don't like the EC icon" responses.

**Reply by Lineflyer** [reply]
Sounds reasonable. 
We already have that collection of feedbacks, which might point into certain directions or give some direction about what to optimize first.

**Reply by ztNFny** [reply]
After beta lets wait a week for people to get used to the icons, then do the survey.
Can we include html in the survey to include screenshots?

**Reply by Lineflyer** [reply]
If we use Google Forms the possibilities are rather limited.
Try it yourself whats possible: https://docs.google.com/forms/u/0/

**Comment by ztNFny** [comment]
Below my proposal (all images not final). I wouldn't ask for every icon individually but only the ones where we got negative feedback so far, instead a questions about the style in general. I'd also like to use to opportunity to get some insight into how people are using cgeo - I'm sure there's some more stuff we should/could add to that list to help us in development, e.g. regarding bottom nav.

Questions regarding the new icon style:
- How do you like the new icons overall? (image of all icons from https://ztnfny.github.io/cgeoicons/index.html) [1-5]
- From the initial beta feedback we've identified and tuned some icons
- Which Multi icon do you like better? (image) [3 waypoints, square waypoints, ]
- Do you want to give us some additional feedback/input on the Multi icon? What do you like about it, what do you not like, and importantly, WHY is that? [textarea]
- Which Earthcache icon do you like better? (image) [earth, satellite, ]
- Do you want to give us some additional feedback/input on the Earthcache icon? What do you like about it, what do you not like, and importantly, WHY is that?[textarea]
- Transparent vs marker for "visited waypoints"

Questions regarding your use of c:geo:
- Do you use compact icons? (mixed screenshot of map) [yes (only), yes (mixed with normal icons), no (only normal icons)]
- How many different websites (Geocaching.com, Opencaching, ...) are you using in c:geo? [1, 2, 3 or more]
- How do you handle modifying coordinates, e.g. when solving a mystery or playing a multi? [Set as corrected coordinates in c:geo or on the website (moves the cache marker on map), Add a "final" waypoint in c:geo, Write it in Note only, Other] (checkboxes)
- Do you mark waypoints as visited when playing a multi?

![2021-10-29_15-42-03](https://user-images.githubusercontent.com/1258173/139445684-da3d1de0-342c-426f-a0c7-480a656c8325.png)

**Timing:**
I'm still working on the multi icon, would like some more time to continue that. Question is whether we do another beta then or just go for the vote.
EC icon I'm not working on anymore

**Reply by ztNFny** [reply]
What I'm still thinking about is the vote for more than 2 icons. Whether to make it a matrix vote (see above) or 2 "vs"-votes (e.g. for multi: 2 boxes vs waypoint style + waypoint style 1 vs waypoint style 2 with options "A, both ok, B") - maybe the later gives a better picture.

**Comment by ztNFny** [comment]
The survey will be sent only to beta users or everyone? Should be beta only imho - others won't know what we're talking about with icons.

**Reply by Lineflyer** [reply]
Only to beta users.

BTW:
I am AFK until 02.11. and probably not answer many threads here by now.

**Comment by Lineflyer** [comment]
I can invest some time tomorrow afternoon or evening for this.

@ztNFny 
Did you already prepare the shown survey draft in some tool, or shall I start hacking it into Google forms (unless you found a better tool)?

I would try to avoid "free text" answers wherever possible. It has proven better for evaluation if we have multiple choice. Google forms allows multiple paths though (so in case of a specific answer you can branch down into a more details multiple choice to refine the feedback). Otherwise you have to manually scan all the free text (remember we have up to 10k beta users (the last survey was answered by short below 1k users)).

Other comments regarding your draft (in case you already have it in a tool and we can use it):
- Questions as posted above are good. The multi should stay in with the options, shall we ask for the earth after all? I think we can stick with the current earthcache or we add a basic feedback mulitple-choice for all icons just to see if there is any icon especially very disliked by users (to see if the earth cache sticks out in a negative way).
- In direct discussion many fellow cachers said, that the new icons look worse. After showing them the direct comparison (on your icons page) most of them immediately liked the new ones more. What I want to say: For the "do you like the icons overall" question I would insert a picture with old ans new in direct comparison.
- Instead of "different website you are using" I would phrase "different geocaching services..." or "different geocaching provider"
- Google Forms is not supporting multi language unfortunately, but I would still enter english and german in parallel at least as we have a rather large german user base and in the last survey quite some replied "Ich verstehe nichts"/ "Bitte deutsch".

**Reply by ztNFny** [reply]
> Did you already prepare the shown survey draft in some tool

no, only entered some in Google Forms to see what it's capable off. Didn't find a GDPR compliant solution that wouldn't require self-hosting.

> I would try to avoid "free text" answers wherever possible. It has proven better for evaluation if we have multiple choice.

The more I think about it the less use I see coming out of a survey, and it's getting even smaller for 1-5 answers.
Like if 20% of users say they don't like the icons, what do we do with that? From the feedback here on Github and more so inside the dev team the new design is well liked and I doubt we'll go and undo that.
I'm not even sure it makes sense to ask about the icons in general at all. Maybe for icon option A vs B, but other than that ...

Maybe we can get together for some discussion tomorrow evening, like a jitsi? I'm available after 6

**Reply by Lineflyer** [reply]
> Maybe we can get together for some discussion tomorrow evening, like a jitsi? I'm available after 6

Yeah, lets just have a chat.
I did not finish my time plan for tomorrow though..I will be most likely able to make 8pm for sure.
I will fire up my Jisti at https://home.droescher.eu:9443 by then.

**Reply by Lineflyer** [reply]
I will be online around 8pm on https://home.droescher.eu:9443/icon-survey

**Reply by ztNFny** [reply]
While preparing screenshots for the survey I stumbled upon the old "has corrected" flag - if someone says the new one is hard to see show them this 🤣
![image](https://user-images.githubusercontent.com/1258173/139947861-180de91b-0aa0-4e15-b428-5fb3588a5e45.png)

**Reply by ztNFny** [reply]
Overview
![2021-11-02_22-03-23](https://user-images.githubusercontent.com/1258173/139951310-03bc0084-3320-450a-bb64-be2f70a3c24e.png)

compact icons:
![Screenshot_1635883683](https://user-images.githubusercontent.com/1258173/139945954-0784be29-857c-4a9e-b62c-dede9d7aa45f.png)
Multi options:
![2021-11-02_21-43-12](https://user-images.githubusercontent.com/1258173/139949631-a0563c46-2e03-4ddb-b481-4e4ca8ca7a0a.png)
Multi options detail:
![2021-11-02_21-50-55](https://user-images.githubusercontent.com/1258173/139949799-af729c07-42f0-423d-9747-23d12c125fcf.png)
Earth options:
![2021-11-02_21-51-09](https://user-images.githubusercontent.com/1258173/139949815-926a5704-7945-4ece-8c3e-8ac441764b84.png)

**Reply by Lineflyer** [reply]
Thanks!! I will prepare the survey and post the link here for review next days.

One very minor thing:
In the first picture the old icons have their pins at the bottom but the new ones are missing them. If you find time and there is a possibility I would like the new ones with pins as well in order not to confuse users which might think, the new one will do no longer have these (although they should know from using the beta, that they are still there).

**Reply by ztNFny** [reply]
> In the first picture the old icons have their pins at the bottom but the new ones are missing them.
![image](https://user-images.githubusercontent.com/1258173/140642114-b28024d5-925a-4086-b337-0d8136233b67.png)

**Comment by Lineflyer** [comment]
Early draft of the survey, feedback for corrections and improvements are welcome:
https://docs.google.com/forms/d/1v7xvZ8K8dNieBnJq6FEvnscQR98CZa9Yha3yrAwOLPE/

Feel free to answer it, I can reset it before publishing.

Regarding the timing I agreed with @ztNFny today:
We will publish another beta (see dedicated issue), wait for at least over one weekend and only then post the survey link as notification to beta users.

**Reply by eddiemuc** [reply]
I did the survey and I liked it very much, thanks for all the work! Especially good I find that both German and English are available.

Following feedback from me:
* All questions should have an option "I don't know", e.g. usage of main screen functions (standard survey practice)
* At the ver end there should be a text field allowing the user additional comments (standard practice in all surveys I know)
* We should ask near the end some generic (and optional) questions to classify users better. E.g.:
  * How often do you go geocaching / how many Geocaches have you found yet? (Categories only of course)
  * In which country are you located?
  * Are you a GC premium member?
  * Is cgeo your main geocaching app / which other apps are you using for caching?
* Maybe a hint why we specifically ask for opinion on multi + earth cache icon (and not for all the rest)?
* Geocaching service count: we should clarify whether we interpret Adventure Labs as its own service (as it is technically implemented) or not. What about making this a multi-select question instead with options GC.com, labs, oc-plattforms, others (e.g. ec)? (Btw: don't we know this already from statistics anyway?)
* On the question how final coords are handled, I suggest reformating third answer to: "I note it in my personal note and rely on c:Geo's auto-waypoint-creation Feature" (that would be my personal answer too :-))
* Number of lists: add an option "more than 20"
* Add a question: "number of caches stored" (with categories). Herr the options "I don't know" and "I don't care" are especially meaningful

I would seriously consider doing same survey for whole user base (not Just Beta) once we are released

**Reply by ztNFny** [reply]
> All questions should have an option "I don't know", e.g. usage of main screen functions

shouldn't be everywhere but only where it's needed and we tried to phrase the quesions to avoid that. For the main screen functions you can't know how often you use them.

> What about making this a multi-select question instead with options GC.com, labs, oc-plattforms, others (e.g. ec)? (Btw: don't we know this already from statistics anyway?)

Your suggestion would give us the statistics we already have, the percentage of users using connector X. What we don't know is how many connectors are being used, hence the question.
Re ALC, that's why it's asking for geocaching sites, ALC are on GC.com.

**Reply by eddiemuc** [reply]
> > All questions should have an option "I don't know", e.g. usage of main screen functions
> 
> shouldn't be everywhere but only where it's needed and we tried to phrase the quesions to avoid that. For the main screen functions you can't know how often you use them.

It is standard practice in all survey to make **all** questions optional **always**. You don't want users to abort survey just because they don't want to answer one of the questions (because this is what users will do if they have no other choice) or give a "random" answer (which will mess up the survey's result).
In order to "tweak" the user in giving an answer anyway, practice is not to make the question itself optional but to allow for an "opt-out" choice (which however has to be selected by user explicitely"). I would not deviate from common survey practices. But we can name it to something like "No answer / keine Angabe" maybe?

**Comment by eddiemuc** [comment]
>Your suggestion would give us the statistics we already have, the percentage of users using connector X. What we don't know is how many connectors are being used, hence the question.

Well, I would then rather integrate this in the current statistics than to ask in a survey :-)

> Re ALC, that's why it's asking for geocaching sites, ALC are on GC.com.

Ok, in this case my feedback is that this is not clearly formulated (at least I didn't understand it). Maybe use something like "geocacing.com (including Labs connector)"?

**Comment by Lineflyer** [comment]
> At the ver end there should be a text field allowing the user additional comments (standard practice in all surveys I know)

I could add that, however it will produce quite some hundreds to thousand free text comments...One might only be able to quickly scan them for common topics.
We did such in the last survey and ended up with many help requests we could never answer as the survey is anonymous. So I think it wont be worth it.

> We should ask near the end some generic (and optional) questions to classify users better. E.g.[...]

What help/guidance would you expect from such questions. 
And the result is somewhat predictable if we ask beta user as those might be the more "extreme" cachers anyway.

> Maybe a hint why we specifically ask for opinion on multi + earth cache icon (and not for all the rest)?

I discussed this with @ztNFny in our yesterdays call and we focused on asking which icon is better instead of much text and reasoning on purpose.


> Geocaching service count: we should clarify whether we interpret Adventure Labs as its own service (as it is technically implemented) or not. What about making this a multi-select question instead with options GC.com, labs, oc-plattforms, others (e.g. ec)? (Btw: don't we know this already from statistics anyway?)

I don't think many users are aware of our connector model but rather on the website they see at the bottom of our (old) main menu. It might more confuse users explaining to not count Adventure Labs as most of them don't see it as a dedicated connector after all.
Even if some of them do, the percentage will not impact the general survey a lot.

> On the question how final coords are handled, I suggest reformating third answer to: "I note it in my personal note and rely on c:Geo's auto-waypoint-creation Feature" (that would be my personal answer too :-))

I changed it to  "I will only note it down in my personal note and let c:geo create a waypoint from it / Ich schreibe sie nur in die persönliche Notiz und lasse c:geo einen Wegpunkt daraus erstellen"

> Number of lists: add an option "more than 20"

Done.

> Add a question: "number of caches stored" (with categories). 

Done with <100, 100-500, >500 

> All questions should have an option "I don't know", e.g. usage of main screen functions

We can decide between two options:
- Mark all the questions as non mandatory, so the user can skip without answering
- Add such "Don't know" option
What do you think?

**Reply by ztNFny** [reply]
> Done with <100, 100-500, >500

That seems quite low? Especially as we're also asking about > 20 lists. I have >20k caches stored.

**Reply by Lineflyer** [reply]
> That seems quite low? Especially as we're also asking about > 20 lists. I have >20k caches stored.

You are right.
I added a zero: <1000, 1000-5000, >5000 

I guess its a question to get an indication only...or is there a purpose/change/bug behind?

**Reply by ztNFny** [reply]
We discussed that question yesterday and decided to leave it out. The result will be pretty much aligned with number of lists.

**Reply by Lineflyer** [reply]
mmh..indeed I would not like to add significantly more questions. I personally hate long surveys and more like those short snapshots with a few questions only.
Especially we should take into consideration, that on Google forms we receive the results only if the users completes the survey.

**Reply by ztNFny** [reply]
Let's leave it out then? I think one of these question (number of lists / number of caches) is enough.

> I changed it to "I will only note it down in my personal note and let c:geo create a waypoint from it / Ich schreibe sie nur in die persönliche Notiz und lasse c:geo einen Wegpunkt daraus erstellen"

Not everyone using note has the "create waypoints" feature enabled though ...

**Reply by Lineflyer** [reply]
> Not everyone using note has the "create waypoints" feature enabled though ...

Correct, I will just put the second half of the sentence(s) into braces then.

**Reply by eddiemuc** [reply]
>> We should ask near the end some generic (and optional) questions to classify users better. E.g.[...]

> What help/guidance would you expect from such questions.
> And the result is somewhat predictable if we ask beta user as those might be the more "extreme" cachers anyway.

I also suggested to give the survey out to all users later (once this is released), and I think it would be helpful to differentiate e.g. the views of experienced users to e.g. sporadic users.

**Reply by eddiemuc** [reply]
> Add such "Don't know" option What do you think?

I suggest adding "not specified / keine Angabe"

**Reply by Lineflyer** [reply]
> I also suggested to give the survey out to all users later (once this is released), and I think it would be helpful to differentiate e.g. the views of experienced users to e.g. sporadic users.

For that one we could use such questions...still it will IMHO hardly deliver a representative result...sporadic users might more rare take the survey...but we could try in such follow-up survey.

> I suggest adding "not specified / keine Angabe"

So, leave all items as mandatory but add such option where applicable?

**Reply by eddiemuc** [reply]
> > I also suggested to give the survey out to all users later (once this is released), and I think it would be helpful to differentiate e.g. the views of experienced users to e.g. sporadic users.
> 
> For that one we could use such questions...still it will IMHO hardly deliver a representative result...sporadic users might more rare take the survey...but we could try in such follow-up survey.

That's exactly why I would add the questions (and already in beta, don't guess what users might answer, just ask them). So later we could clearly separate these groups and make separate stats for them (the one with less exp users might have less answers, but that would be ok. And maybe the numbers will be surprisingly high).

The real bias you always get with these things is that you naturally get more answers from people liking on online surveys than from people not liking them :-) And this might be independent from their usage rate of c:geo


> > I suggest adding "not specified / keine Angabe"
> 
> So, leave all items as mandatory but add such option where applicable?

That's my suggestion, yes. Earlier in this thread however there was also the opinion raised that this should not be done (by @ztNFny iirc)

**Comment by Lineflyer** [comment]
I updated the form according to your replies: https://docs.google.com/forms/d/e/1FAIpQLSc8tauoL4KAvWCf18aGB0cfP86cVKlYrzt16MDIo6dzRXrVeQ/viewform

Any more feedback?

The plan would be to publish the form via notification to the beta version after this weekend.

**Reply by Lineflyer** [reply]
I thought back and forth about any free text and finally added on "short free text" optional question at the end of chapter 1:
![image](https://user-images.githubusercontent.com/949669/140641214-052c5698-8bff-4354-8116-b96de559cfdc.png)

**Reply by bekuno** [reply]
On page one - the last entry is "Do you have an".
This is not finished, isn't it?

For "How do you handle the final cache coordinates e.g. when solving a mystery or playing a multi?"
The answer 3 is similar to 2 - create a waypoint or let them create from a note is a taste of use case. Finally the waypoint is set to "Final".
Therefore I would remove answer 3 (maybe change 2 to "I just add a "final" waypoint in c:geo (manually or by a user note)"

On page 3 (feedback) I think that a free text area could bring some detailed information.

**Reply by ztNFny** [reply]
> The answer 3 is similar to 2 [...] Finally the waypoint is set to "Final".

That's only true if the user writes in the note "Final: " AND has the "extract waypoints from note" feature enabled.
I'd prefer to stay with the original wording for that question (remove the stuff about auto-extract in brackets).

> On page 3 (feedback) I think that a free text area could bring some detailed information.

Lineflyer convinced me of not having free-text answers. If people really want to tell us something they can use the support form. And someone has to read all those free-text answers - and in case people mistake it for a support form (which happened during the last survey) they expect an answer, which we can't provide via the survey.

@Lineflyer Maybe add a link to the support site as last "question"? Like "If you have further feedback feel free to contact us at ..."

**Reply by Lineflyer** [reply]
> On page one - the last entry is "Do you have an".
This is not finished, isn't it?

I don't find it. Can you post a screenshot?

> I'd prefer to stay with the original wording for that question (remove the stuff about auto-extract in brackets).

Sounds like a compromise. The answers felt to long/complex anyways.
Done.

> Lineflyer convinced me of not having free-text answers.

Yes, those will be hard to analyze. Still I added one "short reply" free text for chapter 1 but I would not like to do this for the last chapter...

> Maybe add a link to the support site as last "question"? Like "If you have further feedback feel free to contact us at ..."

There should be our support mail mentioned on the page after the survey completed, but seems I cannot use HTML there.

The all icon picture has also been updated.

**Reply by ztNFny** [reply]
Oh, i only just now saw the introductary text. Imho that doesn't fit well. 2/3 pages of the survey are usage questions.

Headline: "c:geo - Beta Feedback - Icons". Both "icon" and "beta" are only partially true. How about just "c:geo user survey"?
Intro text on page 1 should not just be an intro text for that page (as on page 2 and 3) but is read as an intro text for everything. So I'd rearrange it a bit:

"With this survey we would like to get your input on some features in c:geo we're working on and how you use c:geo. This will help us with future development of c:geo.

The first part of the survey focuses on the new cache and waypoint icons that are available in the beta version you are currently using."

**Reply by bekuno** [reply]
> That's only true if the user writes in the note "Final: " AND has the "extract waypoints from note" feature enabled.

Sure. But if I wish that this is my final, then I change this manually afterwards.
The given question in the survey is: "How do you handle the **final** cache coordinates".
And point 3 says "I will **only** note it down in my personal note". This is too hard to decide. (point 1 **usually **, point 2 **just**)

Maybe a multi selection or answers with 1 to 5 point for every method is better.

**Reply by bekuno** [reply]
> > On page one - the last entry is "Do you have an".
> > This is not finished, isn't it?
> 
> I don't find it. Can you post a screenshot?

Too late. Now the "Anything else ..." is shown there. Fine.

**Reply by Lineflyer** [reply]
> Headline: "c:geo - Beta Feedback - Icons". Both "icon" and "beta" are only partially true. How about just "c:geo user survey"?

I renamed it to "c:geo - Beta Feedback #2" to stay in the scheme we used for the first one

> Intro text on page 1 should not just be an intro text 

I changed it. Shall we still mention we we changed the icons after all? I removed that for now.

> And point 3 says "I will only note it down in my personal note". This is too hard to decide. (point 1 **usually **, point 2 just)

IMHO a valid point. I will look into the wording?
@ztNFny Do you think we should allow multiple answers for that one?

**Reply by ztNFny** [reply]
> But if I wish that this is my final, then I change this manually afterwards.

But that's really answer 2 then, you add it as waypoint.

> Shall we still mention we we changed the icons after all?

If they didn't notice till now ... ;-)

> Do you think we should allow multiple answers for that one?

The statistics might get useful with MC answers as it's not clear how it's grouped, so that'd have to be manually evaluated, thus I'm against that.
All poossible answers are:
- Add as waypoint with corrected
- Add as waypoint without corrected
- Note only
- Corrected + Note
- Waypoint without corrected + Note
- Note with auto-extraction and naming the waypoints in the note ("Final" prefix)
- Note with auto-extraction, then going into the waypoints tab

That gets way to complex!

The original aim of the question was to figure out how users figure out if a cache has a final and where it's stored for which there are 3 options: Corrected, Final Waypoint, neither (e.g. a text in note and use "Go to")
I think if we just add back the word "only" in the answer to option 3 ("Write it in Note ONLY") it should be clear enough

**Reply by Lineflyer** [reply]
> > Shall we still mention we we changed the icons after all?
> 
> If they didn't notice till now ... ;-)

Sorry...typo, I wanted to write:  Shall we still mention **why** we changed the icons after all?

> That gets way to complex!

Thats probably right.

> I think if we just add back the word "only" in the answer to option 3 ("Write it in Note ONLY") it should be clear enough

OK. Agreed from my side.
Will change in next minutes.

**Reply by ztNFny** [reply]
> Do you think we should allow multiple answers for that one?

If we make it MC then I'd have to answer all 3 boxes, because
- I **always** set corrected
- For multis I add waypoints as I go (setting the latest one as corrected)
- I document the stages of the multi in note
That answer just as MC options (all 3 checked) would be completly useless and not representative of my usage at all.

Suggested rewording:
How do you MOST COMMONLY handle the final cache coordinates e.g. when solving a mystery or playing a multi?
- I set them as corrected coordinates in c:geo or on the website
- I add a "final" waypoint in c:geo 
- I will write them down in personal note ONLY
- None of the above

**Reply by bekuno** [reply]
> 
> 
> > But if I wish that this is my final, then I change this manually afterwards.
> 
> But that's really answer 2 then, you add it as waypoint.

This was my argument - answer 3 is in this wording useless.
At least the wording is terrible: **usually**, **just**, **only**

> > Do you think we should allow multiple answers for that one?
> 
> The statistics might get useful with MC answers as it's not clear how it's grouped, so that'd have to be manually evaluated, thus I'm against that. All possible answers are:
> ...
> That gets way to complex!

I don't understand your argumentation.
We have already a list of answers. 
It would be fine to make them (with a little wording change) to multi selection.

> The original aim of the question was to figure out how users figure out if a cache has a final and where it's stored for which there are 3 options: Corrected, Final Waypoint, neither (e.g. a text in note and use "Go to") I think if we just add back the word "only" in the answer to option 3 ("Write it in Note ONLY") it should be clear enough

If I write coords in a note then I have normally a waypoint. Thats the same as creating a waypoint without making them to a final.

So we have:
- use corrected coordinates for real cache coordinates
- use a Final waypoint (create them or from note)
- use waypoints without setting as Final or corrected (create them or from note)
- do not create waypoints

**Reply by bekuno** [reply]
>     * For multis I add waypoints as I go (setting the latest one as corrected)

That is a valid extra point.

>       That answer just as MC options (all 3 checked) would be completly useless and not representative of my usage at all.

Thats why I came with the idea of a 1 to 5 point answer for the presented methods.

> Suggested rewording: How do you MOST COMMONLY handle the final cache coordinates e.g. when solving a mystery or playing a multi?
> 
>     * I set them as corrected coordinates in c:geo or on the website
>     * I add a "final" waypoint in c:geo
>     * I will write them down in personal note ONLY
>     * None of the above

Sounds good.
Lets go this way.

**Reply by ztNFny** [reply]
> We have already a list of answers. It would be fine to make them (with a little wording change) to multi selection.

No, evaluating a MC question is completly different. Let's say we have 2 response, one with my answers as given above: Corrected+Final+Note. A second with: None
Evaluation: 25% Corrected, 25% Final, 25% Note, 25% None.
Reality in current cgeo implementation: 50% see the corrected marker, 50% don't document anything and thus nothing happens.
Does that sound correct to you?

**Reply by bekuno** [reply]
> No, evaluating a MC question is completly different. Let's say we have 2 response, one with my answers as given above: Corrected+Final+Note. A second with: None
> Evaluation: 25% Corrected, 25% Final, 25% Note, 25% None.
> Reality in current cgeo implementation: 50% see the corrected marker, 50% don't document anything and thus nothing happens.
> Does that sound correct to you?

Yes.
But you have to divide by number of users, the result is: 50% Corrected, 50% Final, 50% Note, 50% None.
With a bigger number of answers the result **per user** cold be 60% Corrected, 60% Final, 30% Note, 5% None.

**Comment by Lineflyer** [comment]
Another question:
If I push the notification to the start screen I might use any of the icons in https://github.com/cgeo/cgeo/tree/master/main/res/drawable-mdpi

Any preference?

For the notification text I would also use the en/de mix like this:
"Tap here to participate in a survey for this beta version!  Tippe hier um an einer Umfrage zur aktuellen Beta-Version teilzunehmen!"

**Reply by Lineflyer** [reply]
First try (on developer builds):
![image](https://user-images.githubusercontent.com/949669/140643441-160f517b-15b6-4478-85d1-d0cfdf769c9b.png)

**Reply by ztNFny** [reply]
Maybe add a sentence to the introducation why it's english and german (and not many other languages).
(Is it just that pretty much all devs are german or is also that there's a larger userbase in germany?)

**Reply by fm-sys** [reply]
> Is it just that pretty much all devs are german or is also that there's a larger userbase in germany?

Germany is indeed country no. 1 when counting cgeo users, followed by US and France IIRC...

**Reply by ztNFny** [reply]
> First try (on developer builds):

Missing a "," after "Tippe hier"

And maybe separate english german more with a slash? "... version! / Tippe hier, ..."

**Reply by Lineflyer** [reply]
> Missing a "," after "Tippe hier"
> And maybe separate english german more with a slash? "... version! / Tippe hier, ..."

Thanks, will include those changes.
Shall we already start with the survey? IMHO we could as the whole beta phase is running long right now with only minor icon changes lately.

**Comment by Lineflyer** [comment]
Survey is online for ```(kind == "rc" && versionCode >= 20211103) || (kind =="nightly" && versionCode >= 30211108)```

As you can see I added the recent nightlies, just knowing that some "extreme-beta users" use those regularly and should be included into the survey. The only side effect for nightly users is, that the update notification will be overruled for that time.

I will keep it active until after next weekend to reach most users and then share the result link (into the closed group).

**Comment by Lineflyer** [comment]
I ended the survey (disable the link).
For those of you, who want to dive into it (or missed our todays call), here is a CSV export:
[c_geo - Beta Feedback_2.csv.zip](https://github.com/cgeo/cgeo/files/7534818/c_geo.-.Beta.Feedback_2.csv.zip)


And some screenshots:

![image](https://user-images.githubusercontent.com/949669/141701387-511c0472-f428-462e-af5c-5fc44d77164a.png)

![image](https://user-images.githubusercontent.com/949669/141701396-7afbbb6d-f029-4c18-a368-e64642fa4867.png)
A is the current implementation (in beta), B is the two box version

![image](https://user-images.githubusercontent.com/949669/141701405-56164c93-3b11-4b61-8c87-446af47c26e8.png)
A is the current globe icon, B the "wifi signal" ;)
![image](https://user-images.githubusercontent.com/949669/141701413-9d997d8b-2682-4666-8da4-7d222191345e.png)

![image](https://user-images.githubusercontent.com/949669/141701424-1a5f4886-dc3e-48d7-be75-0c8f37cf1546.png)

![image](https://user-images.githubusercontent.com/949669/141701435-b3316d4d-59fa-48e4-8c40-07922d628cbc.png)

![image](https://user-images.githubusercontent.com/949669/141701448-1cfbf931-39dc-48ba-a9a2-f2ea89441d4b.png)

![image](https://user-images.githubusercontent.com/949669/141701462-a7611675-c35a-471c-baf5-ecdd7ce036e0.png)

![image](https://user-images.githubusercontent.com/949669/141701468-9b09fcc9-9e7c-4dfb-a017-a751ed9133e8.png)

![image](https://user-images.githubusercontent.com/949669/141701496-b5c35568-8184-419c-936c-4dc751b38444.png)

![image](https://user-images.githubusercontent.com/949669/141701533-0e9c0328-c9e1-43d4-bb3a-76863223f767.png)

**Reply by ztNFny** [reply]
Went through all the free text feedback, a lot of unrelated stuff in there. Here's some extracts:


Icon feedback:
- custom can't color is too close to multi - true, they are close. I used the cgeo background color. Should we change that? The connector has a different icon shape than multis though
- There's a handful of comments like this that I don't understand: "I prefer the old icon as it is more consistent with the officials Geocaching icon."
The new icons are a lot closer to GC than the old ones were though. If you "find" such a user please ask what they mean exactly.

Ideas / general stuff
- labs don't get marked as found
- own icon for solved Mystery (-> solved by https://github.com/cgeo/cgeo/pull/12083)
- "An Android auto extension.   Just to see caches as you travel"
- "Macht den Karten- und Routenupdater stumm. Das dauernde Gepiepse (pro Objekt 2 Töne!) nervt ungemein"
- icon scaling (make them larger, make them smaller, add a size between dot and normal)

Stuff already supported but the users don't know (how) - we should improve UI or documentation there (it seems the documentation isn't known either):
- what is the compact icon mode?
- connector icon styles ("Ich habe noch nicht herausgefunden, wieso derselbe Cachetyp (genauer ein Multi) manchmal mit einem runden und manchmal mit einem eckigen Symbol dargestellt wird.")
- "When you create usericon on map, there is one step more, than could be. To make easy red dot I have to click for group and then again for dot."
- separate icon for bonus/challenge caches (-> this could be solved with list icons)

**Reply by Lineflyer** [reply]
> "Macht den Karten- und Routenupdater stumm. Das dauernde Gepiepse (pro Objekt 2 Töne!) nervt ungemein"

Such user (or this user) also contacted us on support mail. I asked for details but did not get an answer. 

>  (it seems the documentation isn't known either):

We have a link to it in our About c:geo page and on homepage and FAQ. Some users however don't like to read documentation. But I would also not like to overfreight c:geo with popups or other helpers informing about why something works as it does. Not all features can be designed self explanatory IMHO.

**Reply by fm-sys** [reply]
>>"Macht den Karten- und Routenupdater stumm. Das dauernde Gepiepse (pro Objekt 2 Töne!) nervt ungemein"
>
>Such user (or this user) also contacted us on support mail. I asked for details but did not get an answer.

There will be an android notification message shown (with Sound if device is not muted) as soon as the import is complete. This is normal/intended...
